### PR TITLE
AI Default Law Change to Crewsimov

### LINKED ISSUE #1070: The AI uses Asimov, not Crewsimov

### PR DESCRIPTION
**Pull request by Kmsxkuse**
:cl:
fix: Changes Default AI laws from Asimov to Crewsimov.
/:cl:

Hopefully Fixes #1070

### CODE REVIEW

File: code/datums/ai_laws.dm [hunk 1]
```diff
@@ -216,8 +216,8 @@
 /datum/ai_laws/proc/set_laws_config()
 	switch(config.default_laws)
 		if(0)
-			add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-			add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
+			add_inherent_law("You may not injure a crewmembers or, through inaction, allow a crewmembers to come to harm.")
```
Comment (ike709) [inline code/datums/ai_laws.dm]: Grammatical error. Correct version:

You may not injure crewmembers or, through inaction, allow crewmembers to come to harm.

File: code/datums/ai_laws.dm [hunk 2]
```diff
@@ -205,8 +205,8 @@
 		add_inherent_law(line)
 	if(!inherent.len) //Failsafe to prevent lawless AIs being created.
 		log_law("AI created with empty custom laws, laws set to Asimov. Please check silicon_laws.txt.")
```
Comment (ike709) [inline code/datums/ai_laws.dm]: Replace "Asimov" with "Crewsimov"

File: code/datums/ai_laws.dm [hunk 3]
```diff
@@ -204,9 +204,9 @@
 
 		add_inherent_law(line)
 	if(!inherent.len) //Failsafe to prevent lawless AIs being created.
-		log_law("AI created with empty custom laws, laws set to Asimov. Please check silicon_laws.txt.")
-		add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-		add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
+		log_law("AI created with empty custom laws, laws set to Crewsimov. Please check silicon_laws.txt.")
+		add_inherent_law("You may not injure crewmembers or, through inaction, allow crewmembers to come to harm.")
+		add_inherent_law("You must obey orders given to you by crewmembers, except where such orders would conflict with the First Law.")
```
Comment (ninjanomnom) [inline code/datums/ai_laws.dm]: You're hardcoding something which is meant to be set by configs again. This code only goes off if it fails to read silicon_laws.txt or if that file is empty.

Comment (Kmsxkuse) [inline code/datums/ai_laws.dm]: Hmmm, right. I'll see if changing one of the configs two to might do it. I didnt want to fiddle around with it but.... sure.

Comment (Kmsxkuse) [inline code/datums/ai_laws.dm]: After many tests, I determined that changing the config does work. I think we should keep the changes to default anyways so they'll work as failsafes should something go wrong.

File: config/game_options.txt [hunk 4]
```diff
@@ -311,7 +311,7 @@ SEC_START_BRIG
 ## Set to 1 for "custom", silicons will start with the custom laws defined in silicon_laws.txt. (If silicon_laws.txt is empty, the AI will spawn with asimov and Custom boards will auto-delete.)
 ## Set to 2 for "random", silicons will start with a random lawset picked from random laws specified below.
 ## Set to 3 for "weighted random", using values in "silicon_weights.txt", a law will be selected, with weights specifed in that file.
-DEFAULT_LAWS 0
+DEFAULT_LAWS 1
```
Comment (ninjanomnom) [inline config/game_options.txt]: No need to change default configs for this. Once this is changed back then this is fine code-wise but I don't feel we should be changing default laws in the code.

**Pull request closed without merging**